cvat/apps/engine/migrations/0075_project_uuid_initial_migration_and_annotation_guide_is_public.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.2.1 on 2023-09-04 16:54
```

Comment:
Probably, you can put all 3 migrations in 1 file, as they do just 1 thing.

Reply:
I'm not sure because I followed the official [documentation](https://docs.djangoproject.com/en/3.2/howto/writing-migrations/#migrations-that-add-unique-fields) for this moment and it is advised to split it into 3 migrations. Maybe this is required to guarantee the sequence of execution.